aws/resource_aws_ses_receipt_rule.go

```diff
@@ -683,9 +683,15 @@ func buildReceiptRule(d *schema.ResourceData, meta interface{}) *ses.ReceiptRule
 			elem := element.(map[string]interface{})
 
 			s3Action := &ses.S3Action{
-				BucketName:      aws.String(elem["bucket_name"].(string)),
-				KmsKeyArn:       aws.String(elem["kms_key_arn"].(string)),
```

Comment:
Can we please add an acceptance test which fails on master and passes with this patch to ensure we don't hit this regression in the future?

Reply:
sure.